roles/control_node/tasks/20_all_workshop.yml

```diff
@@ -6,6 +6,18 @@
     owner: "{{ username }}"
     group: "{{ username }}"
 
+- name: Copy ansible.cfg to a custom directory
+  file:
+    path: "/home/{{ username }}/ansible_config"
```

Comment:
why do we need this?

Reply:
selinux won't allow mounting the `/home/student1/` to the container but we can mount a directory, so we need to create a directory first that will be mounted to `/etc/ansible`

Reply:
How about just moving it to /etc/ansible/anisble.cfg? Is there a reason it needs a custom dir?

Reply:
This way both the approaches can be used, running with ansible-playbook(with ansible.cfg in home dir) and running with ansible-navigator(with the mount approach), 
Not sure if that is the case, but having an option is always good :)

Reply:
@anshulbehl I stand corrected but doesn't `ansible-playbook` also look for the config at /etc/ansible/anisble.cfg?

If you've tested that and it's the best way to do it.  Perfect

Reply:
I am trying to target precedence here, ansible-playbook will pick ~/.ansible.cfg as its still there, ansible-navigator will pick as the `~/.ansible.cfg` is copied to custom dir that's mounted to /etc/ansible for navigator specifically

Reply:
I guess I am confused or don't understand, it seems like a much simpler fixed based on your comments to change the file->
```
roles/control_node/tasks/20_all_workshop.yml
```

first task->
```
---
- name: Install ansible.cfg in home directory
  template:
    src: ansible.cfg.j2
    dest: "/home/{{ username }}/.ansible.cfg"
    owner: "{{ username }}"
    group: "{{ username }}"
```

to 

```
---
- name: Install ansible.cfg in home directory
  template:
    src: ansible.cfg.j2
    dest: "/home/etc/ansible.cfg"
    owner: "{{ username }}"
    group: "{{ username }}"
```

Reply:
Should be fixed now!!, Please review